hs_core/page_processors.py

```diff
@@ -186,7 +186,9 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
                    'bag_url': bag_url,
                    'show_content_files': show_content_files,
                    'discoverable': discoverable,
-                   'is_mine': content_model.is_mine
+                   'is_mine': content_model.is_mine,
+                   'res_id': content_model.short_id,
```

Comment:
@pkdash is it needed to pass res_id page context to page template? ```cm.short_id``` is already used in several places in genericresource.html, so I don't think there is a need to pass res_id page context. Perhaps is_mine can be retrieved directly as well in template genericresource.html by doing ```cm.is_mine```